UnityProject/Assets/Scripts/Items/Implants/Organs/Heart.cs

```diff
@@ -74,6 +76,21 @@ public override void ImplantPeriodicUpdate()
 		DoHeartBeat(RelatedPart.HealthMaster);
 	}
 
+	public override void InternalDamageLogic()
+	{
+		base.InternalDamageLogic();
+		if(RelatedPart.CurrentInternalBleedingDamage > 50 && alarmedForInternalBleeding == false)
+		{
+			Chat.AddActionMsgToChat(RelatedPart.HealthMaster.gameObject, 
+			"You feel a sharp pain in your chest!", $"{RelatedPart.HealthMaster.PlayerScriptOwner.visibleName} holds their chest in pain!");
```

Comment:
the player's heart may not necessarily always be in the chest

Reply:
That's why we have counter measures that stop these things when they're not inside a body.

Reply:
I meant as an if someone implanted a heart in someone's head

Reply:
fixed